packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -4515,6 +4518,7 @@ export interface VideoGalleryProps {
 
 // @public
 export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
+    contentSharingStream?: HTMLElement;
```

Comment:
Have you considered not having a new field and just reusing screenShareStream? i.e. have the selector fill the fields of the screensharestream instead of a new contentSharingStream.
When designing the interface for UI components, they should be agnostic of ACS. In its current state, does `contentSharingStream` function exactly the same as a screenshare stream? (as well as only showing one of them if both are supplied?)

Reply:
ContentSharingStream will be mostly like HTML,  but screen share is same as video stream, raw data. It is kind of different. 

contentSharingStream does not has isMirror, isAvailable, id, etc. 

We will only show one stream, web calling SDK should handle this logic.

Reply:
Can you mention more the difference between a content stream and a screen share stream?
Screenshare streams, and video streams, are just a `renderElement` which is an `HTMLElement`.
From my understanding, ContentSharingStream is the same - an uninteractive stream that is provided as an HTMLElement?

If contentShare is more than just a "stream" - i.e. it has things like interactable controls -- we should be avoiding the name `stream` and we will need more properties than just an HTMLElement.

It is odd currently that screenshare takes in things like isMirrored because the screenshare stream is never mirrored by design...